tests/input/matcher_rules.yaml

```diff
@@ -60,4 +60,18 @@ rules:
         - oio:hasNarrowSynonym
     postconditions:
       predicate_id: skos:narrowMatch
-      weight: 2.0
\ No newline at end of file
+      weight: 2.0
+
+  - synonymizer:
```

Comment:
I would like to think a bit more about how to include preprocessing pipelines in the data model. Here is my instinct.

1. Preprocessing pipelines are pipelines that transform the label/synonym from one state to another
2. There are some built-in pipelines (Caseing, Lemmatisation, Whitespace etc). These are named in the matcher_rules_data_model.
3. Some of them are default (Caseing, Whitespace, etc), some optional
4. We have a language to define _new_ preprocessing pipelines (add `id`)
5. In the preconditions section, we can add `subject_preprocessing`, `object_preprocessing` (a list of Preprocessing pipelines)

So if we fold in a "synonymiser pipeline", we can decide to do that for one rule, but not for another (and adjust, for example, the weight in the postconditions). What do you think?